static/js/settings_account.js

```diff
@@ -146,7 +146,7 @@ exports.add_custom_profile_fields_to_settings = function () {
                 }
             }
 
-            if (value !== undefined && value.length > 0) {
+            if (value) {
```

Comment:
Can you explain why this is correct?  I'd put it in its own commit regardless, but to me it looks like a mistake (at least, `value.length` is a fine thing to do given `value` is not undefined...).

Reply:
yeah, it was confusing. Convert it to its own commit to handle `null` values.